### PR TITLE
improved unit test coverage in Tools

### PR DESCRIPTION
**Pull request by michalkaptur**
Clean up and test 3 methods from Tools

## Description

Tools::humanReadableFileSize
Tools::isHex
Tools::isBase64

## Types of changes

- ✅ improved UT coverage

## Checklist:

- ✅ I have read the **CONTRIBUTING** document. **[REQUIRED]**
- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**
- ✅ I have compiled and verified my code with `-DWITH_ASAN=ON`. **[REQUIRED]**
- ✅ My change requires a change to the documentation and I have updated it accordingly.
- ✅ I have added tests to cover my changes.

### CODE REVIEW

**Comment by TheZ3ro** [comment]
Seems that there is a problem with the build @phoerious 

Anyway it's nice to see PR for unit tests :smile:

File: tests/TestTools.h [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (C) 2015 Enrico Mariotti <enricomariotti@yahoo.it>
```
[the remark this answers is not in the document]
Comment (michalkaptur) [reply tests/TestTools.h]: ✅

File: tests/TestTools.cpp [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/*
+ *  Copyright (C) 2015 Enrico Mariotti <enricomariotti@yahoo.it>
```
[the remark this answers is not in the document]
Comment (michalkaptur) [reply tests/TestTools.cpp]: ✅

**Comment by michalkaptur** [comment]
@TheZ3ro I've fixed the tests, there was an invalid assumption on decimal separator.

**Comment by michalkaptur** [comment]
@TheZ3ro Do you have the permission to merge the PR?

**Comment by michalkaptur** [comment]
@phoerious PR rebased and squashed

**Comment by phoerious** [comment]
Thanks!

**Pull request merged**